code/game/objects/items/sticker.dm

```diff
@@ -0,0 +1,138 @@
+/obj/item/sticker
+	name = "sticker"
+	desc = "A sticker with some strong adhesive on the back, sticks to stuff!"
+	item_flags = NOBLUDGEON | XENOMORPH_HOLDABLE //funny
+	resistance_flags = FLAMMABLE
+	icon = 'icons/obj/stickers.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	throw_range = 3
+	vis_flags = VIS_INHERIT_DIR | VIS_INHERIT_PLANE | VIS_INHERIT_LAYER
+	///The overlay we apply to things we stick to
+	var/mutable_appearance/sticker_overlay
+	///A list of icon_states to pick an icon_state on Initialize, provided it is not null.
+	var/list/icon_states
+	///The thing we are attached to
+	var/atom/attached
+	///The turf our COMSIG_TURF_EXPOSE is registered to, so we can unregister it later.
+	var/turf/signal_turf
+
+/obj/item/sticker/Initialize(mapload)
+	. = ..()
+	if(icon_states)
+		icon_state = pick(icon_states)
+	pixel_y = rand(-3,3)
+	pixel_x = rand(-3,3)
+
+/obj/item/sticker/afterattack(atom/target, mob/living/user, prox, params)
+	. = ..()
+	if(!prox)
+		return
+	if(!isliving(target) && !isobj(target) && !isturf(target))
+		return
+	user.visible_message(span_notice("[user] sticks [src] to [target]!"),span_notice("You stick [src] to [target]!"))
+	var/list/parameters = params2list(params)
+	var/divided_size = world.icon_size / 2
+	var/py = text2num(parameters["icon-y"]) - divided_size
+	var/px = text2num(parameters["icon-x"]) - divided_size
+	. |= AFTERATTACK_PROCESSED_ITEM
+	stick(target,user,px,py)
+
+///Sticks this sticker to the target, with the pixel offsets being px and py.
+/obj/item/sticker/proc/stick(atom/target, mob/living/user, px,py)
+	sticker_overlay = mutable_appearance(icon, icon_state , layer = target.layer + 1, appearance_flags = RESET_COLOR | PIXEL_SCALE)
+	sticker_overlay.pixel_x = px
+	sticker_overlay.pixel_y = py
+	target.add_overlay(sticker_overlay)
+	attached = target
+	register_signals(user)
+	moveToNullspace()
+
+///Makes this sticker move from nullspace and cut the overlay from the object it is attached to, silent for no visible message.
+/obj/item/sticker/proc/peel(datum/source, silent = FALSE)
+	SIGNAL_HANDLER
+	if(!attached)
+		return
+	attached.cut_overlay(sticker_overlay)
+	sticker_overlay = null
+	forceMove(attached.drop_location())
+	if(!silent)
+		visible_message(span_notice("[src] falls off [attached]."))
```

Comment:
Balloon alert

Reply:
Will that be a bit long for a balloon alert? Tweaking the text here might be nessary too

Reply:
Considering the pixels are the visual feedback, I think we can get away with no chat/balloon here at all